tutorials/katacoda/thanos/2-lts/step4.md

```diff
@@ -0,0 +1,40 @@
+# Step 4 - Thanos Compactor
+
+In this step, we will install Thanos Compactor which applies the compaction procedure of the Prometheus 2.0 storage engine to block data in object storage.
+
+Before, moving forward, let's take a closer look at what the `Compactor` component does:
+
+## Compactor
+
+The `Compactor` is an essential component that operates on a single object storage bucket to compact, down-sample, apply retention, to the TSDB blocks held inside, thus, making queries on historical data more efficient. It creates aggregates of old metrics (based upon the rules).
+
+It is also responsible for downsampling of data, performing 5m downsampling after 40 hours, and 1h downsampling after 10 days.
+
+If you want to know more about Thanos Compactor, jump [here](https://thanos.io/tip/components/compact.md/).
+
+**Note**: Thanos Compactor is mandatory if you use object storage otherwise Thanos Store Gateway will be too slow without using a compactor.
+
+## Deploying Thanos Compactor
+
+Click below snippet to start the Compactor.
+
+```
+docker run -d --net=host --rm \
+    -v $(pwd)/bucket_storage.yml:/etc/prometheus/bucket_storage.yml \
+    --name thanos-compact \
+    quay.io/thanos/thanos:v0.15.0 \
+    compact \
+    --data-dir             /prometheus \
+    --objstore.config-file /etc/prometheus/bucket_storage.yml \
+    --http-address         0.0.0.0:19092
+```{{execute}}
+
```

Comment:
To show if this work we can look on Compact HTTP port BUCKET VIEWER! :heart:

Reply:
The port isn't working for me weird it, can you check too please?

Reply:
`working weird` ... is it new HTTP status code? (: 

I assume you mean 404 .You need to go to `<port>/loaded`. Will describe this (:

Reply:
(we need to explain that as well plus UI)